<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerEMSegmentEditRegistrationParametersStep</class>
 <widget class="qSlicerEMSegmentWorkflowWidgetStep" name="qSlicerEMSegmentEditRegistrationParametersStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>163</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="AtlasToInputRegistrationParametersGroupBox">
     <property name="title">
      <string>Atlas-to-Input Registration Parameters</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="AtlasScansToInputChannelsLayout"/>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="AffineRegistrationLabel">
          <property name="text">
           <string>Affine Registration:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="AffineRegistrationComboBox">
          <property name="toolTip">
           <string>Select affine registration.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="DeformableRegistrationLabel">
          <property name="text">
           <string>Deformable Registration:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="DeformableRegistrationComboBox">
          <property name="toolTip">
           <string>Select deformable registration.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="InterpolationLabel">
          <property name="text">
           <string>Interpolation:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="InterpolationComboBox">
          <property name="toolTip">
           <string>Select interpolation type.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkWorkflowWidgetStep</class>
   <extends>QWidget</extends>
   <header>ctkWorkflowWidgetStep.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerEMSegmentWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerEMSegmentWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerEMSegmentWorkflowWidgetStep</class>
   <extends>ctkWorkflowWidgetStep</extends>
   <header>qSlicerEMSegmentWorkflowWidgetStep.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
